### DISCUSSION TITLE
Post build script?

### DISCUSSION THREAD

**Discussion by trampas** [microsoft/vscode-cmake-tools#3393] [General]
I have cmake kits setup to compile code for x86 and ARM target, is there a way to have a post build option in cmake-kits.json for the arm target which will run script to scp file to target for testing?

**Comment by gcampbell-msft** [comment]
We do have tasks that you can set up in `tasks.json` which allow you to configure and build there. Then, if you want to have another task that `depends` on those tasks, you could set it up that way. Something like this (as an example): 
![image](https://github.com/user-attachments/assets/2d81a0e3-ba34-496c-90a7-aacb5eb3b892)

We do not support this as a definition from cmake-kits.json.
